src/main/kotlin/com/ongdev/blog/api/services/impl/CategoryServiceImpl.kt

```diff
@@ -0,0 +1,66 @@
+package com.ongdev.blog.api.services.impl
+
+import com.ongdev.blog.api.models.dtos.requests.CategoryCreationRequest
+import com.ongdev.blog.api.models.dtos.requests.CategoryUpdateRequest
+import com.ongdev.blog.api.models.dtos.responses.CategoryCreationResponse
+import com.ongdev.blog.api.models.dtos.responses.CategoryListResponse
+import com.ongdev.blog.api.models.mapToCategory
+import com.ongdev.blog.api.models.repositories.CategoryRepository
+import com.ongdev.blog.api.models.toCategoryCreationResponse
+import com.ongdev.blog.api.models.toCategoryEntity
+import com.ongdev.blog.api.services.interfaces.CategoryService
+import org.springframework.stereotype.Service
+import java.util.*
+import kotlin.RuntimeException
+
+@Service
+class CategoryServiceImpl(val categoryRepository: CategoryRepository): CategoryService {
+
+    override fun getCategoryById(id: String): CategoryCreationResponse {
+        val category = (categoryRepository.findById(UUID.fromString(id))).orElse(null);
```

Comment:
Nếu không tìm được thì em throw exception nhe

Reply:
@nancynguyen14701 chỗ orElse, thì em sử dụng orThrow 1 cái not found ex là ok nha, và anh thấy là cái dấu ngoặc bọc cái findById là dư thì phải